doc/manual/syntax.md

```diff
@@ -227,7 +225,7 @@ For those cases, Nickel uses symbolic strings:
 }
 ```
 
-This example is an excerpt of a Nix configuration written in Nickel emulating
+This example is an excerpt of a Nix configuration written in Nickel, emulating
 Nix string context. Lines 4 to 8 define a symbolic string. Values `inputs.gcc`,
```

Comment:
```suggestion
Nix string contexts. Lines 4 to 8 define a symbolic string. Values `inputs.gcc`,
```

Reply:
That something that I'm always wondering. For example Shea's blogpost, context is singular (https://shealevy.com/blog/2018/08/05/understanding-nixs-string-context/). I know I've written it plural elsewhere, and probably a mix of the two :sweat_smile:

Reply:
I guess plural can't be wrong, at least. Let's go with your suggestion for now.

Reply:
Well, if it weren't a plural here it would at least need an article. E.g. emulating *a* Nix string context. Or it could be Nix's string context. But that seems weird to me, because every string in Nix carries a context, so there's many of them.